java/src/jmri/configurexml/swing/Bundle.java

```diff
@@ -0,0 +1,98 @@
+package jmri.configurexml.swing;
+
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import java.util.Locale;
+import javax.annotation.CheckReturnValue;
+import javax.annotation.CheckForNull;
+import javax.annotation.ParametersAreNonnullByDefault;
+
+@ParametersAreNonnullByDefault
+@CheckReturnValue
+@SuppressFBWarnings(value = "NM_SAME_SIMPLE_NAME_AS_SUPERCLASS", justification = "Desired pattern is repeated class names with package-level access to members")
+
+@javax.annotation.concurrent.Immutable
+
+/**
+ * Provides standard access for resource bundles in a package.
+ *
+ * Convention is to provide a subclass of this name in each package, working off
+ * the local resource bundle name.
+ *
+ * @author Bob Jacobsen Copyright (C) 2012
+ * @author Mark Underwood Copyright (C) 2015
+ */
+public class Bundle extends jmri.Bundle {
+
+    @CheckForNull
+    private static final String name = "jmri.configurexml.Bundle";  // NOI18N
```

Comment:
It would be better if the Bundle.properties file is moved to `jmri.configurexml.swing` so that the Bundle class and the properties file are in the same package.